hopr/hopr-lib/src/config.rs

```diff
@@ -52,6 +57,12 @@ pub struct Chain {
     #[serde(default = "just_true")]
     #[default = true]
     pub check_unrealized_balance: bool,
+    #[serde(default = "just_false")]
+    #[default = false]
+    pub keep_logs: bool,
+    #[serde(default = "just_false")]
+    #[default = false]
+    pub fast_sync: bool,
```

Comment:
@coderabbitai Please create an issue for the mileston 3.0 to remove this functionality (configurability of fast_sync and log keeping) to have it on by default in 3.0.